exercises/practice/space-age/space_age.f90

```diff
@@ -0,0 +1,14 @@
+
+module space_age
+  implicit none
+contains
+
+  double precision function age_in_years(planet, seconds)
+    character(len=*) :: planet
+    double precision :: seconds
+    
+    age_in_years = 0.d0
```

Comment:
What do you think of this? (Is it too verbose, or would you see it as "better"?)

```suggestion
    character(len=*), intent(in) :: planet
    double precision, intent(in) :: seconds
    double precision, intent(out) :: age_in_years = 0.d0
```

Reply:
in production code we always use `intent`, just like `const ` was an afterthought in c but now widely used.

Reply:
Got it - Since "fluency" is the goal for Exercism, I think it'll be good to keep all the example solutions as close to "production" as possible. (We can always iterate and respond to feedback, of course)

I'll take a quick look over existing exercises and see if I can add this to more stubs/examples too

Reply:
Fluency does not equate to production though.  Because fluency is the abilitiy to express ones self, this does not mean production or acedamia style code, it means a fluency of the language, the ability to express oneself in the language.

Reply:
@kotp That's a good point. "Production" implies a lot more, esp ideas of fault tolerance, optimizations, logging, etc, that go way outside the scope of fluency.

I think at least with the `intent(in)` I'd always want to see it from a student, unless modifying a function's input is critical to the exercise for some reason. (So something like the same reasons that I'd want to see students steer away mutable parameters in general)

That said, some googling leads me to believe it might vary by compiler whether this is actually enforced or not